docs/defining-maintainers.md

```diff
@@ -0,0 +1,37 @@
+# Defining L3AF project maintainers
+
+## Finding all the L3AF projects
+
+You can see all of the repositories associated with L3AF by navigating to the top-level [L3AF organization on GitHub](https://github.com/l3af-project).
+
+Typically a L3AF sub-project is represented by a single repository (repo). For instance, the Specs and Architecture project is represented in the [l3af-arch](https://github.com/l3af-project/l3af-arch) repo. In the future it's possible that a single project may require multiple repos. The contents of this document still hold true in that case (and if you find that's wrong, please [open and issue](https://github.com/l3af-project/governance/issues/new/choose) and we'll fix it).
```

Comment:
```suggestion
Typically a L3AF project is represented by a single repository (repo). For instance, the Specs and Architecture project is represented in the [l3af-arch](https://github.com/l3af-project/l3af-arch) repo. In the future it's possible that a single project may require multiple repos. The contents of this document still hold true in that case (and if you find that's wrong, please [open an issue](https://github.com/l3af-project/governance/issues/new/choose) and we'll fix it).
```
Also recommend splitting into multiple source lines to make git diffs easier over time since github doesn't deal well with really long paragraphs.

Reply:
All changes should be in latest update